content/blog/pki-maze.md

```diff
@@ -0,0 +1,72 @@
++++
+title = "Designing a Permanent Personal Identity"
+date = 2019-11-20
```

Comment:
Should be 2019-11-27 if going out tomorrow.

Reply:
good call, will update accordingly when all reviews + edits are addressed

Reply:
Full name OKed